src/js/edu-benefits/5490/config/form.js

```diff
@@ -34,6 +47,10 @@ const formConfig = {
   confirmation: ConfirmationPage,
   title: 'Update your Education Benefits',
   subTitle: 'Form 22-5490',
+  defaultDefinitions: {
+    educationType: educationType.schema,
```

Comment:
I'm tempted to suggest that we always use `defaultDefinitions`, actually. Might be simpler. We can talk about this in the meeting today.